crates/tauri-runtime-wry/src/lib.rs

```diff
@@ -2388,6 +2388,12 @@ pub struct WindowWrapper {
   focused_webview: Arc<Mutex<Option<String>>>,
 }
 
+impl WindowWrapper {
+  pub fn label(&self) -> String {
+    self.label.clone()
+  }
```

Comment:
```suggestion
  pub fn label(&self) -> &str {
    &self.label
  }
```

Don't really care much, but I think it should probably be something like this

Reply:
Agreed. Will update my plugin's downstream dependencies based on this and commit.